doc/persistence_fixture.py

```diff
@@ -0,0 +1,10 @@
+"""Fixture module to skip the persistence doctest with python 2.6."""
+
+from nose import SkipTest
+from joblib import _compat
+
+
+def setup_module(module):
+    """Setup module."""
+    if _compat.PY26:
+        raise SkipTest("Skipping persistence doctest in Python 2.6")
```

Comment:
Can you add a comment to say that GzipFile can not be used as a context manager in Python 2.6 or some better explanation.